### ISSUE TITLE
Support for Sendfox API

### ISSUE THREAD

**Issue by jonbstrong** [kraftbj/genesis-enews-extended#147]
It would be great if there were instructions on how to integrate Genesis Enews with Sendfox!
I've had a shot, but it doesn't seem to be working on my end.

**Comment by lorenam83** [comment]
Hi Jon! Did you figured out how to do the integration? Having the same issue :(

**Comment by jonbstrong** [comment]
Nope, just gave up and used something else instead I'm afraid ¯\_(ツ)_/¯

**Comment by kraftbj** [comment]
Closing. The plugin's direction is deliberately a generic form-action widget rather than per-vendor API integrations (the recent 2.3.0 release removed the legacy FeedBurner and MailPoet 2 integrations for the same reason). Sendfox's standard form HTML works with the generic widget today.
